gcovr/tests/test_gcovr.py

```diff
@@ -57,6 +57,17 @@
 CC = os.path.split(env["CC"])[1]
 IS_CLANG = True if CC.startswith("clang") else False
 
+IS_WINDOWS = platform.system() == "Windows"
+if IS_WINDOWS:
+    import win32api
+    import string
+
+    used_drives = win32api.GetLogicalDriveStrings().split("\0")
+    free_drives = list(set([c for c in string.ascii_uppercase]) - set(used_drives))
```

Comment:
(Style) The `set()` constructor should accept any iterable, not just a list. This should be equivalent:

```suggestion
    free_drives = list(set(string.ascii_uppercase) - set(used_drives))
```